cli/cmd/addon_download_test.go

```diff
@@ -16,6 +17,11 @@ import (
 )
 
 func TestAddonDownload(t *testing.T) {
+	cqDir := t.TempDir()
+	logFileName := path.Join(cqDir, "cloudquery.log")
+	t.Cleanup(func() {
+		CloseLogFile()
+	})
```

Comment:
This seems very formulaic and used in nearly all of the tests, do we want to have a single helper function that just returns a list that can be appended to the args array

```
func setupArgs(t *testing.T) []string{
	cqDir := t.TempDir()
	logFileName := path.Join(cqDir, "cloudquery.log")
	t.Cleanup(func() {
		CloseLogFile()
	})
	return []string{"--cq-dir", cqDir, "--log-file-name", logFileName}
 }

Reply:
Yes we can 👍 Let me follow up with a refactor commit